ProcessMaker/Http/Controllers/Auth/LoginController.php

```diff
@@ -59,7 +59,7 @@ public function showLoginForm()
         // clear cookie to avoid an issue when logout SLO and then try to login with simple PM login form
         \Cookie::queue(\Cookie::forget(config('session.cookie')));
         // cookie required here because SSO redirect resets the session
-        $cookie = cookie('processmaker_intended', redirect()->intended()->getTargetUrl(), 10, '/');
+        $cookie = cookie('processmaker_intended', redirect()->intended()->getTargetUrl(), 10, null, null, true, true, false, 'none');
```

Comment:
@nolanpro is there a reason why you changed the cookie path from '/' to null?

Reply:
@caleeli It's not needed. The default set by config/session.php is '/' so it's the same thing. It's better to use the default values.

Reply:
@nolanpro According to the https://www.php.net/manual/en/function.setcookie.php, the default path for the cookies is "_the current directory that the cookie is being set in_". But yes! in this case the directory of the `/login` page is `/` and that is the same.

On the other hand, 🤔 I understand that the `config/session.php` set the parameters for the cookie `processmaker_session`:

![image](https://user-images.githubusercontent.com/8028650/217035505-fdd89cc2-957b-4e83-9222-593af023bc74.png)